english/alchemy/alchemy.md

```diff
@@ -0,0 +1,7 @@
+You take off your backpack and pull out your trusty acetylene torch. You place the first marshmallow in a crucible and apply the flame. Chanting an incantation that has been passed down through generations of alchemists, you stare into the glowing crucible as the marshmallow melts into a cauldron of orange flame. As the fiery light coalesces into the shape of a coin at the bottom of the crucible, you turn off the flame. You remove a mysterious small wooden box from your backpack, open it, and remove a ceramic coin mold from inside. After placing the mold in front of your feet, you carefully pour the molten metal into the mold.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.